client/src/components/Form/Elements/FormText.vue

```diff
@@ -117,13 +117,15 @@ export default {
             return this.area || this.multiple;
         },
         style() {
-            return this.color ? { ...this.styleObj, color: this.color } : this.styleObj;
+            return this.color
+                ? {
+                      color: this.color,
+                      "border-color": this.color,
+                  }
+                : null;
```

Comment:
This now restores the rendering of workflow parameters e.g. as in:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/2105447/209848511-d1864f08-46ea-4e88-a10d-544b7c5d3fa8.png">